src/features/auth/user.route.js

```diff
@@ -31,6 +32,23 @@ router.get('/users/:id', (req, res) => {
 
 router.post('/', AuthMiddleware.validateEmail, UserController.register);
 router.post('/login', UserController.login);
+//google routes
+router.get('/google',passport.authenticate('google',{ scope:['profile']}))
```

Comment:
Let's scope the email too. We can use it to check if the same email wants to access the system using a different method of registration they used at first.ie oAuth or Password. if it's the same email, we should use the method that was used at first.